test/sql/storage/store_group_order_all.test

```diff
@@ -0,0 +1,46 @@
+# name: test/sql/storage/store_group_order_all.test
+# description: Test storage of GROUP BY/ORDER BY ALL
+# group: [storage]
+
+# load the DB from disk
+load __TEST_DIR__/view_storage.db
+
+statement ok
+PRAGMA disable_checkpoint_on_shutdown
+
+statement ok
+PRAGMA wal_autocheckpoint='1TB';
+
+statement ok
+CREATE TABLE integers(g integer, i integer);
+
+statement ok
+INSERT INTO integers values (0, 1), (0, 2), (1, 3), (1, NULL);
+
+statement ok
+CREATE VIEW v1 AS SELECT g, i, g%2, SUM(i), SUM(g) FROM integers GROUP BY ALL ORDER BY ALL
+
+query IIIII nosort v1
+SELECT * FROM v1
+----
```

Comment:
I'm missing something here - why are these results empty?

Reply:
The result checker here uses the labels (`v1` and `v2`) to verify that the result does not change after restarting. The result is empty because when labels are present the result is ignored and instead results with the same label are compared.

Reply:
Oh duh I knew that. Sorry!